mlrun/run.py

```diff
@@ -201,13 +201,13 @@ def get_or_create_ctx(
     rundb: str = "",
     project: str = "",
     upload_artifacts=False,
-    labels: Optional[dict] = None,
```

Comment:
this needs to be properly deprecated before removing though not exactly sure I understand the reason you want remove it

Reply:
Why I want to remove it:

1. It was added by us by mistake in 1.5 and is not used.
2. The implementation is faulty (as we discussed) - it adds the labels after saving the context in the DB.
Run the following in Jupyter:
```py
import mlrun

ctx = mlrun.get_or_create_ctx("test-label", labels={"kind": "serving"})
print(ctx.to_yaml())
```
The UI shows:
![image](https://github.com/user-attachments/assets/6d6dab4e-b73b-45b0-ba5c-b302b0716611)
The Jupyter client prints (truncated):
![image](https://github.com/user-attachments/assets/4341d6c8-d578-418c-89bc-3a55733e1dc3)

This is a bug.

Reply:
Regarding proper deprecation, since it's relatively new and was not requested, I think no one has used it.
But we can do it... in this case, a fix may be better.

Reply:
I agree probably no one has used it though we do declare 2 minor versions BC. I understand it's annoying and I agree would be best to just delete but I still think we need to uphold our statements.

Reply:
Done in 322febb